packages/bsky/tests/seeds/client.ts

```diff
@@ -139,8 +139,14 @@ export class SeedClient {
           displayName,
           description,
           avatar: avatarBlob,
+          labels: selfLabels
+            ? {
+                $type: 'com.atproto.label.defs#selfLabels',
+                values: selfLabels.map((val) => ({ val })),
```

Comment:
That isn't required here since the `values` field is a ref and not an open union.  The `labels` field above is an open union, so it need to contain the `$type` 👍 We think this will provide a decent amount of flexibility without it becoming too verbose.